.github/workflows/zxcron-extended-test-suite.yaml

```
@@ -491,6 +518,79 @@ jobs:
           fi
           echo "slack-user-id=${SLACK_USER_ID}" >> "${GITHUB_OUTPUT}"
 
+      - name: Set rootly service name
```

Consistency in names
```suggestion
      - name: Set Rootly Service Name
```